src/item.cpp

```diff
@@ -970,24 +970,30 @@ std::string item::info(std::vector<iteminfo> &info, const iteminfo_query *parts,
+            if( !food_item->rotten() ) {
                 if( food_item->is_fresh() ) {
                     info.emplace_back( "DESCRIPTION", _( "* This food looks as <good>fresh</good> as it can be." ) );
-                } else if( rot_progress >= 0.1 && rot_progress < 0.3 ) {
-                    info.emplace_back( "DESCRIPTION", _( "* This food looks <good>still quite fresh</good>. "
+                } else if( rot_progress >= 0.1 && rot_progress < 0.3 && skilled ) {
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks <good>still quite fresh</good>.  "
                                                          "It's far from becoming old." ) );
-                } else if( rot_progress >= 0.3 && rot_progress < 0.5 ) {
-                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it is reaching its <neutral>midlife</neutral>. "
+                } else if( rot_progress >= 0.3 && rot_progress < 0.5 && skilled ) {
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it is reaching its <neutral>midlife</neutral>.  "
                                                          "It has some time ahead before spoiling." ) );
-                } else if( rot_progress >= 0.5 && rot_progress < 0.7 ) {
-                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it has <neutral>passed its midlife</neutral>. "
+                } else if( rot_progress >= 0.5 && rot_progress < 0.7 && skilled ) {
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it has <neutral>passed its midlife</neutral>.  "
                                                          "Edible, but will go old sooner rather then later." ) );
-                } else if( rot_progress >= 0.7 && rot_progress <= 0.9 ) {
-                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it <bad>will be old soon</bad>. "
+                } else if( rot_progress >= 0.7 && rot_progress < 0.9 && skilled ) {
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks like it <bad>will be old soon</bad>.  "
                                                          "It's now or never, if you plan to use it." ) );
                 } else if( food_item->is_going_bad() ) {
-                    info.emplace_back( "DESCRIPTION", _( "* This food looks <bad>old</bad>. "
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks <bad>old</bad>.  "
                                                          "It's on a brink of becoming inedible." ) );
+                } else { // unskilled and neither fresh nor old
+                    info.emplace_back( "DESCRIPTION", _( "* This food looks <neutral>fine</neutral>.  "
+                                                         "If you were more skilled in cooking or survival "
+                                                         "you might be able to make a better estimation." ) );
```

Comment:
Having just noticed this while doing a merge (not doing a proper code review, just making a personal branch), I'd like to see this logic set up differently:

```C++
if( !food_item->rotten() ) {
    if( food_item->is_fresh() ) { // even the unskilled can see it's fresh!
        //...
    } else if (food_item->is_going_bad() ) { // even the unskilled can see it's old!
        //...
    } else {
        info.emplace_back( "DESCRIPTION", describe_freshness( rot_progress, skilled) );
    }
```

Where `describe_freshness(double, bool)` can go something like:

```C++
if( !skilled ) {
    return _( "* This food looks <neutral>fine</neutral>.  "
              "If you were more skilled [...]");
} else if( rot_progress >= 0.1 && rot_progress < 0.3 ) {
    return _( "* This food looks <good>still quite fresh</good>.[...]" );
// ...and so on
```

This is just my personal preference, not necessarily project style; please correct me if it seems like I'm suggesting something very wrong (in particular, separating the strings into different locations in the file might be confusing to contributors -- it's usually better to have them all in the same place, and it's nice to see them in order, too).